crates/shim-protos/Cargo.toml

```diff
@@ -21,13 +22,17 @@ ttrpc-codegen = { version = "0.3.0", optional = true }
 ctrlc = { version = "3.0", features = ["termination"] }
 log = "0.4"
 simple_logger = { version = "2.0", default-features = false, features = ["stderr"] }
+tokio = { version = "1.8.0", features = ["full"] }
```

Comment:
Should this be optional too?

Reply:
I try to add the option for tokio get an error: 
Caused by:
  dev-dependencies are not allowed to be optional: `tokio`

also find an issue for this
https://github.com/rust-lang/cargo/issues/1596